front/src/modules/scenario/components/AddOrEditScenarioModal.tsx

```diff
@@ -19,8 +19,8 @@ import TextareaSNCF from 'common/BootstrapSNCF/TextareaSNCF';
 import InfraSelectorModal from 'common/InfraSelector/InfraSelectorModal';
 import { osrdEditoastApi, ScenarioCreateForm, ScenarioPatchForm } from 'common/api/osrdEditoastApi';
 import { setFailure, setSuccess } from 'reducers/main';
-import { updateScenarioID } from 'reducers/osrdconf';
-import { getInfraID } from 'reducers/osrdconf/selectors';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done